src/App.js

```diff
@@ -45,14 +46,27 @@ class App extends Component {
   filterVideoByType = (dataVideos) => {
     let trailerVideo = dataVideos.filter(video => video.type ===  "Trailer")
     this.setState({
-      videos:[...this.state.videos, ... trailerVideo]
+      videos:[...this.state.videos, ...trailerVideo]
     })   
   }
 
   returnHome = () => {
-    this.setState( {selectedMovie: null, videos: []} )
+    this.setState( {selectedMovie: null, videos: [], filteredMovies: []} )
   } 
 
+  filterMovies = (searchInput) => {
```

Comment:
I'm thinking to change the name of this Method to be more specific (  filterMovieByTitle), what do you think?